perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -44,6 +44,15 @@ chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 sudo umount gcs
 
+# Mounting gcsfuse for running integration tests for the mountedDirectory flag
+BUCKET_NAME=gcsfuse-integration-tests
+MOUNT_POINT=gcs
+# The VM will itself exit if the gcsfuse mount fails.
```

Comment:
please create a seperate script for running integration tests and use it in continous test and pr test.

Reply:
fixed